### ISSUE TITLE
NTR: Wound culture or tissue sample: Acid-fast bacilli present (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8092]
New term request Wound culture or tissue sample: Acid-fast bacilli present (seidel_190313151521): AFB stain positive  UMLS: 0486278

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000963
- label: Demonstration of acid-fast bacilli in wound tissue
- parents: [Positive infectious agent test (HP:0430030)](https://hpo.jax.org/app/browse/term/HP:0430030)
- pmid: PMID:30725806
- def: Demonstration of acid-fast bacilli by microscopic examination of wound tissue. Acid fastness is a physical property that gives a bacterium the ability to resist decolorization by acids during staining procedures.
- comment: Bacteria displaying acid fastness include: Mycobacterium leprae, M. tuberculosis, M. smegmatis, M. Avium complex, M. kansasii, Nocardia brasiliensis, N. cyriacigeorgica, N. farcinica, and N. nova.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8092
- synonyms:
